wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/inc/class-plugin.php

```diff
@@ -95,6 +95,49 @@ function plugins_loaded() {
 		// Correct `WP_Locale` for variant locales in project lists.
 		add_filter( 'gp_translation_sets_sort', [ $this, 'filter_gp_translation_sets_sort' ] );
 
+		// Add site tour items
+		if ( isset( $_GET['site_tour'] ) && 'test' == $_GET['site_tour'] ) {
```

Comment:
It doesn't load for me because we're overriding `templates/header.php`, so this would work:

```suggestion
		if ( isset( $_GET['site_tour'] ) && 'test' == $_GET['site_tour'] ) {
			gp_enqueue_styles( array( 'gp-base', 'gp-jquery-webui-popover' ) );
			gp_enqueue_scripts( array( 'gp-jquery-webui-popover', 'gp-tour' ) );


```